src/App.js

```diff
@@ -21,31 +21,50 @@ function useApp() {
   return { centralTeams, pacificTeams };
 }
 
+function SortTeamsByRank(teams, year) {
+  const teamsInTheYear = teams.map((team) => {
+    const result = team.results.find((el) => el.year === year);
+    return {
+      id: team.id,
+      name: team.name,
+      founded_year: team.founded_year,
+      league: team.league,
+      result: result,
+    };
```

Comment:
If the `team` has only `id`, `name`, `founded_year` and `league` as its properties, you can write:

```suggestion
    return {
      ...team,
      result,
    };
```

However, in JS world, the snake_case is not common and you could use camelCase.
Not `founded_year`, but `foundedYear`

Reply:
camelize するようにして、`...team`表記にしました。
camelize は axios の setup ファイルなどでやるべきだと思うので、別途対応していきます。 85b17ec69d74a0940ae1683a0d9537a5e5218337